sideboard/internal/imports.py

```diff
@@ -1,23 +1,32 @@
 from __future__ import unicode_literals
 import sys
 import importlib
+from collections import OrderedDict
 from glob import glob
+from itertools import chain
 from os.path import join, isdir, basename
 
 from sideboard.config import config
 
-plugins = {}
 
+plugins = OrderedDict()
+plugin_dirs = OrderedDict()
```

Comment:
I am also caching `plugin_dirs`. This information could be derived from `plugins` by iterating over the modules and calling `os.path.join(module.__file__, '..')` but since we already have the information in the format we want, why not go ahead and cache it now?